main.go

```diff
@@ -152,6 +169,31 @@ func main() {
 
 	l := logs.New(logLevel)
 	clientID := uuid.New().String()
+	
+	resp, err := http.Get("https://api.myip.com/")
```

Comment:
There should be 
```suggestion
	defer resp.Body.Close()
```
After error checking (if I recall correctly you're a c++ dev and it's the way RAII is done for go)

Reply:
Ah, interesting, github even allows you to commit suggestions directly via the web UI. I'll try to check it out right now :)

Reply:
I need to resolve this first to make a proper suggestion here